tools/flang1/flang1exe/symacc.h

```diff
@@ -16,7 +16,7 @@ struct SYM;
 #include "sharedefs.h"
 
 /* Keep in sync with INIT_NAMES_SIZE, in syminidf.h. */
-#define STB_NSIZE 5069
+#define STB_NSIZE 5075
```

Comment:
Please also check if a similar change is needed in `tools/flang2/flang2exe/symacc.h`.

Reply:
The `STB_NSIZE` keeps in sync with `INIT_NAMES_SIZE` (both are 3289) in `build/flang/tools/flang2/utils/symtab/syminidf.h`, so the similar change is unneeded.

Reply:
Thanks for confirming.